cve_bin_tool/version_scanner.py

```diff
@@ -157,10 +150,8 @@ def run_python_package_checkers(self, filename, lines):
         There are no actual checkers.
         The ProductInfo is computed without the help of any checkers from PKG-INFO or METADATA.
         """
-        product = search(
-            r"--generated pattern for cve-bin-tool Name: (.+?) Version:", lines[0]
```

Comment:
the `--generated pattern for cve-bin-tool` is removed since we don't create checkers for python packages now.